src/SmartComponents/DriftTable/modules/operations.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
```

Comment:
Open question: should we move this file to something like src/api.js ?

Reply:
I don't have much knowledge of how to do this or how it all works together, but if we can keep from importing axios in every file by doing this, then I don't see why not.